src/open_inwoner/ssd/xml.py

```diff
@@ -0,0 +1,409 @@
+    specificatiejaaropgave_spec = glom(jaaropgave_spec, "SpecificatieJaarOpgave")
+    jaaropgave = {
+        "regeling": glom(specificatiejaaropgave_spec, "Regeling"),
+        "dienstjaar": glom(specificatiejaaropgave_spec, "Dienstjaar"),
+        "periode": {
+            "key": "Tijdvak",
+            "van": format_date(glom(specificatiejaaropgave_spec, "AangiftePeriodeVan")),
+            "tot": format_date(glom(specificatiejaaropgave_spec, "AangiftePeriodeTot")),
+        },
+        "fiscaalloon": {
+            "key": "Loon loonbelasting / volksverzekeringen",
+            "sign": get_sign(
+                specificatiejaaropgave_spec, "Fiscaalloon.CdPositiefNegatief"
+            ),
+            "value": glom(specificatiejaaropgave_spec, "Fiscaalloon.WaardeBedrag"),
+        },
+        "loonheffing": {
+            "key": "Ingehouden Loonbelasting / Premie volksverzekeringen (loonheffing)",
+            "sign": get_sign(
+                specificatiejaaropgave_spec, "Loonheffing.CdPositiefNegatief"
+            ),
+            "value": glom(specificatiejaaropgave_spec, "Loonheffing.WaardeBedrag"),
+        },
+        # TODO: figure out if this is needed; see below
+        "arbeidskorting": {
+            "key": "Verrekende arbeidskorting",
+            "value": "MYSTERY",
+        },
+        "code_loonbelastingtabel": {
+            "key": "Code loonbelastingtabel",
+            "value": glom(specificatiejaaropgave_spec, "CdPremieVolksverzekering"),
+        },
+        "ingehouden_bijdrage": {
+            "key": "Ingehouden bijdrage Zorgverzekeringswet",
+            "sign": get_sign(
+                specificatiejaaropgave_spec, "IngehoudenPremieZVW.CdPositiefNegatief"
+            ),
+            "value": glom(
+                specificatiejaaropgave_spec, "IngehoudenPremieZVW.WaardeBedrag"
+            ),
+        },
+        "vergoeding_premie_zvw": {
+            "sign": get_sign(
+                specificatiejaaropgave_spec, "VergoedingPremieZVW.CdPositiefNegatief"
+            ),
+            "value": glom(
+                specificatiejaaropgave_spec, "VergoedingPremieZVW.WaardeBedrag"
+            ),
+        },
+        "loon_zorgverzekeringswet": {
+            "key": "Loon Zorgverzekeringswet",
+            "value": calculate_loon_zvw(
+                glom(specificatiejaaropgave_spec, "Fiscaalloon.WaardeBedrag"),
+                glom(specificatiejaaropgave_spec, "VergoedingPremieZVW.WaardeBedrag"),
+            ),
+        },
+        "werkgevers_heffing_premie": {
+            "key": "Werkgeversheffing Zorgverzekeringswet",
+            "sign": get_sign(
+                specificatiejaaropgave_spec,
+                "WerkgeversheffingPremieZVW.CdPositiefNegatief",
+            ),
+            "value": glom(
+                specificatiejaaropgave_spec,
+                "WerkgeversheffingPremieZVW.WaardeBedrag",
+            ),
+        },
+        # TODO: figure out where this is used; relation to `arbeitskorting`?
```

Comment:
Create a task for this, seems like a TODO that still needs to be tackled

Reply:
That's part of what we discussed with Rob and should be resolved when we hear back. I'll create a task with this and the other points.